Capri.Services/DiplomaCard/DrawingGetter.cs

```diff
@@ -0,0 +1,86 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Wordprocessing;
+using System.IO;
+using A = DocumentFormat.OpenXml.Drawing;
+using DW = DocumentFormat.OpenXml.Drawing.Wordprocessing;
+using PIC = DocumentFormat.OpenXml.Drawing.Pictures;
+
+namespace Capri.Services.DiplomaCard
+{
+    public class DrawingGetter: IDrawingGetter
+    {
+        public Drawing getPutLogo(MainDocumentPart mainPart, string imagePath)
+        {
+            ImagePart imagePart = mainPart.AddImagePart(ImagePartType.Png);
+            using (FileStream stream = new FileStream(imagePath, FileMode.Open))
+            {
+                imagePart.FeedData(stream);
+            }
+
+            var image = new Drawing(
+                 new DW.Anchor(
+                     new DW.SimplePosition() { X = 0L, Y = 0L },
```

Comment:
Personally me would prefere another style of writing this, smth. like https://www.stuartwhiteford.com/creating-a-word-document-with-the-open-xml-sdk-20/

But this is not critical, you can leave it as it is.